.gitignore

```diff
@@ -8,3 +8,4 @@ bin-release
 built
 reports
 package-lock.json
+*.DS_Store
```

Comment:
is this a mistake ?

Reply:
Not accidental, no. These files show up in unstaged for me otherwise, am I missing something?

Reply:
@jnatalzia This is fine, but it's good to add that to your global git config/ignore settings https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
👍